packages/teams-js/src/public/app.ts

```diff
@@ -518,6 +519,7 @@ export namespace app {
         authentication.initialize();
         menus.initialize();
         pages.config.initialize();
+        dialog.initialize();
```

Comment:
If a developer never uses a dialog, then the `dialogMessageHandler` will stay registered the whole time for no reason. 
Should we consider registering it only if a dialog is opened?

Reply:
There's no way to confirm that it is in a dialog or not, but the registered handler is non-invasive. Additionally it will only ever use `dialogMessageHandler` if we are in the `tasks` context.